app/assets/stylesheets/application.css

```diff
@@ -128,6 +128,8 @@ form {
   background-color: #f2f2f2;
   padding: 20px;
   width: 60%;
+  margin-left: auto;
+  margin-right: auto;
 }
```

Comment:
Since I was looking at our app in the browser, horizontally centered all forms for more consistent styling (looks like they had headers center-aligned).

Reply:
Awesome!

Reply:
Awesome